### PR TITLE
docs: datetime functions docs

### PR DESCRIPTION
**Pull request by Veeupup**
Signed-off-by: Veeupup <code@tanweime.com>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

add `datetime` function docs in `docs` and `website/docs`

## Changelog

* Improvement
* Documentation

## Related Issues

Part of #4483

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Review by sundy-li — APPROVED** [review]

**Review by BohuTANG — APPROVED** [review]
Great to me

**Comment by BohuTANG** [comment]
```
[ FAIL ] - result differs with:
--- /home/runner/work/databend/databend/tests/suites/0_stateless/06_show/06_0005_show_functions.result	2022-03-29 09:43:48.675274728 +0000
+++ /home/runner/work/databend/databend/tests/suites/0_stateless/06_show/06_0005_show_functions.stdout	2022-03-29 09:45:44.910773322 +0000
@@ -1,8 +1,8 @@
 today	1	0		Returns current date.
-todayofmonth	1	0		
+todayofmonth	1	0		Converts a date or date with time to a UInt8 number containing the number of the day of the month (1-31).
 todayofweek	1	0		Converts a date or date with time to a UInt8 number containing the number of the day of the week (Monday is 1, and Sunday is 7).
 todayofyear	1	0		Converts a date or date with time to a UInt16 number containing the number of the day of the year (1-366).
 today	1	0		Returns current date.
-todayofmonth	1	0		
+todayofmonth	1	0		Converts a date or date with time to a UInt8 number containing the number of the day of the month (1-31).
 todayofweek	1	0		Converts a date or date with time to a UInt8 number containing the number of the day of the week (Monday is 1, and Sunday is 7).
 todayofyear	1	0		Converts a date or date with time to a UInt16 number containing the number of the day of the year (1-366).

```

File: docs/doc/03-reference/02-functions/03-datetime-functions/todayofweek.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## Arguments
+
+| Arguments   | Description |
+| ----------- | ----------- |
+| expr | date16/date32/datetime |
+
+## Return Type
+Datetime object, returns date in “YYYY-MM-DD” format.
```
Comment (b41sh) [inline docs/doc/03-reference/02-functions/03-datetime-functions/todayofweek.md:20]: Should the return value be `Uint8`

File: docs/doc/03-reference/02-functions/03-datetime-functions/tomonday.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+## Arguments
+
+| Arguments   | Description |
+| ----------- | ----------- |
+| expr | date16/date32/datetime |
+
+## Return Type
+Datetime object, returns date in “YYYY-MM-DD” format.
```
Comment (b41sh) [inline docs/doc/03-reference/02-functions/03-datetime-functions/tomonday.md:21]: ditto

File: docs/doc/03-reference/02-functions/03-datetime-functions/todayofweek.md (hunk 1 above)
Comment (Veeupup) [reply docs/doc/03-reference/02-functions/03-datetime-functions/todayofweek.md:20]: yes, thanks

**Pull request merged**